neo4j@.service

```diff
@@ -35,7 +36,6 @@ ExecStart=/bin/sh -c '\
     --env "GRAPHITE_ENABLED=true" \
     --env "GRAPHITE_ADDRESS=graphite.ft.com:2003" \
     --env "GRAPHITE_PREFIX=coco.services.$ENV.neo4j.%i" \
-    --env "NEO4J_dbms_memory_heap_maxSize=8G" \
```

Comment:
I think we still need to pass the environment variable at docker runtime - it'll just be instead of a hardcoded 8gb value, we'll pull the value from etcd.